props/saber_fett263_buttons.h

```diff
@@ -1911,6 +1923,45 @@ SaberFett263Buttons() : PropBase() {}
     sound_library_.SayBool(current_value);
   }
 
+  int GetStyleNumber(int blade_num) {
+    const char* tmp;
+    tmp = current_preset_.GetStyle(blade_num);
+    int style_num = FirstWord(tmp, "builtin") ? atoi(SkipWord(tmp)) : 0;
+    return style_num;
+  }
+
+  void ChangeStyleNumber(int blade_num, int direction) {
+    int num_presets = current_config->num_presets;
+    style_num_ += direction;
```

Comment:
Using style_num_ seems really weird here.
Why not just use a local variable and call GetStyleNumber(blade_num) to fetch it?

Reply:
updated